docs/postgres_2_duckdb.md

```diff
@@ -0,0 +1,38 @@
+# Why would I migrate?
+DuckDB offers a number of advantages in some contexts. Some reasons include
+1. Better performance: In a small job DuckDB is just a file which in the context
+1. Lower dependencies: Unlike Postgres, which requires a dedicated server or virtual machine to run, DuckDB is just a single file that you can run directly on your device. This makes it easier to set up and deploy, and it reduces the need for additional infrastructure and maintenance.
+1. Pushing data to S3/Parquet, if your end goal is to pull data from a tap, and push it to s3/parquet you probably don't need to use a datawarehouse and instead can just run locally 
+
+# How to migrate your Meltano project from Postgres to DuckDB
+1. Install target-duckdb, follow the install steps [here](https://hub.meltano.com/loaders/target-duckdb)
+2. If you're using dbt already in Meltano I'd recommend just updating the pip_url for dbt to `dbt-core~=1.2.0 dbt-duckdb~=1.2.0 duckdb==0.6.1`, this is easier than using the `dbt-duckdb` as you won't have to merge the files that come along with `dbt-duckdb` note these versions will be out of date. Pull the latest versions from the hub 
+    1. You will need to update your dbt `profiles.yml` to the type of `duckdb`, as well as add a `path` in an `output` pointing to your duckdb database.
+Example profiles.yml
```

Comment:
I'm assuming a lot here like that you know what `profiles.yml` is and a bunch of other small details.